--- /usr/share/logstash/vendor/jruby/lib/ruby/shared/mkmf.rb	2017-04-28 18:11:45.000000000 +0000
+++ /usr/share/logstash/vendor/jruby/lib/ruby/shared/mkmf.rb.new	2017-06-02 14:31:42.325422414 +0000
@@ -43,7 +43,8 @@
 RbConfig::MAKEFILE_CONFIG["CFLAGS"] += " $(cflags)"
 RbConfig::MAKEFILE_CONFIG["CPPFLAGS"] += " $(DEFS) $(cppflags)"
 RbConfig::MAKEFILE_CONFIG["CXXFLAGS"] += " $(cflags) $(cxxflags)"
-
+RbConfig::MAKEFILE_CONFIG["CPPFLAGS"] += ' -I/usr/local/rvm/rubies/ruby-1.9.3-p551-dev/include/ruby-1.9.1/x86_64-linux/'
+RbConfig::MAKEFILE_CONFIG['includedir'] = "/usr/local/rvm/rubies/ruby-1.9.3-p551-dev/include/ruby-1.9.1/"
 $topdir     = RbConfig::MAKEFILE_CONFIG['includedir']
 $hdrdir     = File.join($topdir, "ruby")
 $top_srcdir = $topdir
